rfd/0156-notifications.md

```diff
@@ -162,83 +186,87 @@ message ByRoles {
 const (
-   // DefaultInformationalCode is the default code for an informational notification.
-   DefaultInformationalCode = "default-informational"
-   // DefaultWarningCode is the default code for a warning notification.
-   DefaultWarningCode = "default-warning"
+   // DefaultInformationalType is the default type for an informational notification.
+   DefaultInformationalType = "default-informational"
+   // DefaultWarningType is the default type for a warning notification.
+   DefaultWarningType = "default-warning"
 
-   // UserCreatedInformationalCode is the code for informational notifications that were manually created by a user using `tctl`.
-   UserCreatedInformationalCode = "user-created-informational"
-   // UserCreatedWarningCode is the code for warning notifications that were manually created by a user using `tctl`.
-   UserCreatedWarningCode = "user-created-warning"
+   // UserCreatedInformationalType is the type for informational notifications that were manually created by a user using `tctl`.
+   UserCreatedInformationalType = "user-created-informational"
+   // UserCreatedWarningType is the type for warning notifications that were manually created by a user using `tctl`.
+   UserCreatedWarningType = "user-created-warning"
 
-   // AccessRequestPendingCode is the code for the notification of an access request pending review. This is the notification received
+   // AccessRequestPendingType is the type for the notification of an access request pending review. This is the notification received
    // by a user when an access request needs their review.
-   AccessRequestPendingCode = "access-request-pending"
-   // AccessRequestApprovedCode is the code for the notification of a user's access request being approved.
-   AccessRequestApprovedCode = "access-request-approved"
-   // AccessRequestDeniedCode is the code for the notification of a user's access request being denied.
-   AccessRequestDeniedCode = "access-request-denied"
+   AccessRequestPendingType = "access-request-pending"
+   // AccessRequestApprovedType is the type for the notification of a user's access request being approved.
+   AccessRequestApprovedType = "access-request-approved"
+   // AccessRequestDeniedType is the type for the notification of a user's access request being denied.
+   AccessRequestDeniedType = "access-request-denied"
 )
 ```
 
-For example, if a notification's code is `access-request-pending`, the UI will make the notification redirect to the page for reviewing the access request.
+For example, if a notification's type is `access-request-pending`, the UI will make the notification redirect to the page for reviewing the access request.
 
 #### Usage
 
-The notifications service will work primarily by adding, deleting, and fetching notifications from the database. There will also be a watcher which
+The notifications service will work primarily by adding, deleting, and fetching notifications from the backend. There will also be a watcher which
```

Comment:
I think I've asked several times if we can eliminate the watcher to simplify things. Is this just a leftover or are you still planning on needing a watcher?

Reply:
This was a leftover.